feeds/wlan-ap/opensync/patches/37-add-auto-hw_mode.patch

```diff
@@ -0,0 +1,12 @@
+--- a/interfaces/opensync.ovsschema
++++ b/interfaces/opensync.ovsschema
+@@ -1357,7 +1357,8 @@
+                   "11n",
+                   "11ab",
+                   "11ac",
+-                  "11ax"
++                  "11ax",
++                  "auto"
```

Comment:
Did you synchronize this with Mike on the Cloud side?

Reply:
Not yet, I'll talk to him and get that started.

Reply:
@RickSommerville I've talked to Mike about this. But he wants to wait untill it gets merged into pending to make any changes on the cloud side